src/cascadia/LocalTests_TerminalApp/TabTests.cpp

```diff
@@ -430,7 +430,7 @@ namespace TerminalAppLocalTests
         Log::Comment(L"Duplicate the tab, and don't crash");
         result = RunOnUIThread([&page]() {
             page->_DuplicateFocusedTab();
-            VERIFY_ARE_EQUAL(2u, page->_tabs.Size(), L"We should gracefully do nothing here - the profile no longer exists.");
+            VERIFY_ARE_EQUAL(3u, page->_tabs.Size(), L"We should successfully duplicate a tab hosting a deleted profile.");

```

Comment:
huh, that's nice. I'm assuming this doesn't close out an issue on our repo then?

Reply:
Nah. Duplication still isn't _finished_; i referenced the appropriate issue in the PR body.